src/src/components/EventsStore/index.ts

```diff
@@ -101,6 +101,18 @@ export function useEvents(
   );
 }
 
+export function useCachedEvents(): EventsStore | undefined {
+  const cachedEventsStr = window.localStorage.getItem('events');
+  if (!cachedEventsStr) return undefined;
+  const cachedEvents = JSON.parse(cachedEventsStr) as EventsStore;
+  if (!cachedEvents) return undefined;
+  return cachedEvents;
+}
+
+export function setCachedEvents(events: EventsStore): void {
+  window.localStorage.setItem('events', JSON.stringify(events));
```

Comment:
This is not correct. This is accessing Google Chrome's local storage.
Extensions have their own storage that they should use - https://developer.chrome.com/docs/extensions/reference/storage/

The API is quite similar though so should be easy to change

Reply:
I believe this should be good now